package/com.unity.formats.usd/Samples/UsdTimelinePlayable/SetDefaultUSDMesh.cs

```diff
@@ -21,18 +21,25 @@ namespace Unity.Formats.USD.Examples
 {
     /// <summary>Fill in a default USD Asset in case the user didn't choose one
     /// </summary>
-    [ExecuteInEditMode, RequireComponent(typeof(UsdAsset))]
+    [ExecuteAlways, RequireComponent(typeof(UsdAsset))]
     public class SetDefaultUSDMesh : MonoBehaviour
     {
-        private const string K_DEFAULT_MESH = "mesh.usd";
+        private const string K_DEFAULT_MESH = "TimelinePlayableUsd.usd";
 
-        void Start()
+        private void Awake()
         {
-            var asset = GetComponent<UsdAsset>();
-            if (string.IsNullOrEmpty(asset.usdFullPath))
-            {
-                asset.usdFullPath = Path.Combine(PackageUtils.GetCallerRelativeToProjectFolderPath(), K_DEFAULT_MESH);
-            }
+            SetSampleUsdFilePath();
+        }
+
+        void Update()
+        {
+            SetSampleUsdFilePath();
```

Comment:
The file path isn't easily modifiable anyway, and it's demonstrating a bad practice for a user (running something every editor frame, especially something doing a GetComponent and a string manipulation). It would be a very edge case behaviour if a user modified that file path manually, and personally if they did so I think it would be better if it caused an issue (as it would in a real case) instead of silently 'fixed' itself